src/shellstrings.c

```diff
@@ -0,0 +1,105 @@
+
+/* 
+ * String output functions to be used in the shell
+ *
+ * See shellstrings.h for function documentation.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "shellstrings.h"
+
+/* See shellstrings.h */
+void shell_intro() {
```

Comment:
I'm not sure if this is always printed when launching the shell, but please don't do that - only display this if the user presses `h`. People memorize these kinds of tools quick and want to just see a prompt.

Reply:
The condition for this being printed is if the filename is not specified at the command line, at which point this is printed in main_page. Should I opt for a "-h" command line flag instead, and if so, should I maintain status quo of this being printed out if a filename is not given?

Reply:
No flag needed. I'd say just never print it unless the user types the `h` command, which already exists right?